### PR TITLE
Added NXOS provider for radius_global

### PR DESCRIPTION
**Pull request by jonnytpuppet**

### CODE REVIEW

**Comment by jonnytpuppet** [comment]
tests/beaker_tests/radius_global/radius_global_provider_defaults.rb passed in 18.38 seconds
      Test Suite: tests @ 2015-10-21 15:26:54 +0100

File: tests/beaker_tests/radius_global/radius_globallib.rb [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+module RadiusGlobalLib
+  # Group of Constants used in negative tests for radius_global provider.
+  ENSURE_NEGATIVE = 'unknown'
+
+  # A. Methods to create manifests for radius_global Puppet provider test cases.
+
+  # Method to create a manifest for radius_global
+  # @param none [None] No input parameters exist.
+  # @result none [None] Returns no object.
+  def self.create_radius_global_manifest
```
Comment (hunner) [inline tests/beaker_tests/radius_global/radius_globallib.rb]: We should probably have tests for setting the -1'able values to -1. And `key` to `unset`? And checking defaults after creating with just `ensure => present`

**Comment by jonnytpuppet** [comment]
TestCase :: radius_global Resource :: All Attributes Defaults :: PASS
TestCase :: radius_global Resource :: All Attributes Defaults :: End
tests/beaker_tests/radius_global/radius_global_provider_defaults.rb passed in 28.91 seconds

**Comment by jonnytpuppet** [comment]
@hunner there are no removable integer properties in this type. It is also non-ensurable. I've added a test for unsetting the set :)

**Comment by glennmatthews** [comment]
:+1:

**Comment by jonnytpuppet** [comment]
@chrisvanheuveln @glennmatthews can this be merged now that util is merged? :)

**Pull request merged**